climada/util/coordinates.py

```diff
@@ -1155,23 +1158,81 @@ def get_admin1_info(country_names):
         Shape according to Natural Earth.
     """
 
-    if isinstance(country_names, str):
-        country_names = [country_names]
+    if isinstance(countries, (str, int, float)):
+        countries = [countries]
+    if not isinstance(countries, list):
+        raise TypeError("Invalid type for input parameter 'countries'")
     admin1_file = shapereader.natural_earth(resolution='10m',
                                             category='cultural',
                                             name='admin_1_states_provinces')
```

Comment:
All of the analogous methods in this module have `resolution` as a parameter. Probably worth adding here as well.

Separately, they also have `extent` or `bounds` parameter, but I think that might be an issue for another time!

Reply:
for 'admin_1_states_provinces', only one resolution is available (10m), so no choice here unfortunately